SimPEG/EM/TDEM/SrcTDEM.py

```diff
@@ -46,11 +46,16 @@ def evalDeriv(self, time):
 
 class StepOffWaveform(BaseWaveform):
 
+    eps = 1e-9
+
     def __init__(self, offTime=0.):
         BaseWaveform.__init__(self, offTime=offTime, hasInitialFields=True)
 
     def eval(self, time):
-        return 0.
+        if abs(time-0.) < self.eps:
```

Comment:
can we achieve this instead with the `hasInitialFields`? My interpretation of the step-off is that it is off at time zero, but instead has initial fields, so that property is a flag. I am open to change, just curious as to where this is needed.